Eu realizei as tarefas no Pycharm, talvez haja algum bug ao utilizar o intepretador do Python diretamente sobre o arquivo.